src/main/java/sirius/search/properties/StringListProperty.java

```diff
@@ -55,6 +55,9 @@ private StringListProperty(Field field) {
     @Override
     protected Object transformFromSource(Object value) {
         if (!(value instanceof List)) {
+            if (value instanceof String) {
+                return Lists.newArrayList(value);
```

Comment:
pls use `java.util.Collections.singletonList(value)` for (non-measurable) performance reasons